src/app/workspace/twinclass/[twinClassId]/page.tsx

```diff
@@ -9,38 +9,38 @@ import { Tab, TabsLayout } from "@/widgets";
 import { useContext, useEffect } from "react";
 import { TwinClassGeneral, TwinClassStatuses } from "./views";
 
-const tabs: Tab[] = [
-  {
-    key: "general",
-    label: "General",
-    content: <TwinClassGeneral />,
-  },
-  {
-    key: "fields",
-    label: "Fields",
-    content: <TwinClassFields />,
-  },
-  {
-    key: "statuses",
-    label: "Statuses",
-    content: <TwinClassStatuses />,
-  },
-  {
-    key: "twinflows",
-    label: "Twinflows",
-    content: <TwinClassTwinFlows />,
-  },
-  {
-    key: "links",
-    label: "Links",
-    content: <TwinClassLinks />,
-  },
-];
-
 export default function TwinClassPage() {
   const { setBreadcrumbs } = useBreadcrumbs();
   const { twinClass } = useContext(TwinClassContext);
 
+  const tabs: Tab[] = [
+    {
+      key: "general",
+      label: "General",
+      content: <TwinClassGeneral />,
+    },
+    {
+      key: "fields",
+      label: "Fields",
+      content: <TwinClassFields twinClassId={twinClass.id} />,
```

Comment:
lets remove this `twinClassId={twinClass.id}` for timebeing

I'll create a new widget. And will teach you how to use it later